repository_service_tuf_worker/services/keyvault/local.py

```diff
@@ -26,94 +37,167 @@ class LocalKeyVault(IKeyVault):
 
     def __init__(
         self,
-        key_pass: str,
-        key_path: Optional[str] = "online.key",
-        key_type: Optional[str] = "ed25519",
+        path: str,
+        keys: str,
     ):
         """Configuration class for RSTUF Worker LocalKeyVault service.
-        Manages all settings related to the usage of the online key.
+        Manages all settings related to the usage of the online key(s).
 
         Args:
-            key_pass: password to load the online key.
-            key_path: file path of the online key.
-            key_type: cryptography type of the online key.
+            path: path for key vault (used to define the volume)
+            keys: list of online keys to be used. String uses two separators
+               `:` to separate the keys and `,` to separate the field.
+               `<file>,<password>,<type>`
+               file: the file or base64 content (base64|<key body in base64>)
+               password: the key password
+               type: optional, default=ed25519
+               Example:`key1.key,pass1:key2.key,pass2,rsa`
         """
-        self._password: str = key_pass
-        self._path: str = key_path
-        self._type: str = key_type
+        self._path: str = path
+        self._keys: str = keys
 
-        if self._password.startswith("/run/secrets/"):
-            # The user has stored their password using container secrets.
-            with open(self._password) as f:
-                secrets_password = f.read().rstrip("\n")
+    @classmethod
+    def _base64_key(cls, keyvault_path, base64_key) -> str:
+        key_content = base64_key.split("base64|")[1]
+        hash_key = hashlib.blake2b(key_content.encode("utf-8"), digest_size=16)
+        file_name = hash_key.hexdigest()
+        key_path = os.path.join(keyvault_path, file_name)
+        if os.path.isfile(key_path):
+            return key_path
+        else:
+            with open(key_path, "w") as f:
+                f.write(base64.b64decode(key_content).decode())
+
+            return key_path
 
-            self._password = secrets_password
+    @classmethod
+    def _raw_key_parser(cls, keys: str) -> List[LocalKey]:
```

Comment:
Nit: a single short sentence as a doc string will benefit the function readability.